python/kotekan/scripts/polyco_tools.py

```diff
@@ -343,6 +377,7 @@ def update_polyco(
 )
 @click.pass_context
 def import_schedule(ctx, fname, url, tempo_dir, parfile_dir, reference):
+    """NOT IMPLEMENTED YET"""
```

Comment:
What does this mean? We should remove this or prevent people from using it if it doesn't work at all.

Reply:
I could make it just do nothing and print that out instead.

Reply:
You can `raise NotImplementedError`